models/protocol_tests/input/rest-json.json

```diff
@@ -1620,7 +1620,7 @@
             "requestUri": "/path"
           },
           "endpoint":{
-              "hostPrefix": "data-"
+            "hostPrefix": "data-"
```

Comment:
minor nit, was this file intended to be updated? looks like changes were just spacing alignment?